fiat-roles/fiat-roles.gradle

```diff
@@ -15,14 +15,19 @@
  */
 
 dependencies {
-  compile project(':fiat-core')
+  compile project(":fiat-core")
 
   spinnaker.group("retrofitDefault")
 
-  compile spinnaker.dependency('bootWeb')
-  compile spinnaker.dependency('groovy')
+  compile spinnaker.dependency("bootActuator")
+  compile spinnaker.dependency("bootWeb")
+  compile spinnaker.dependency("groovy")
```

Comment:
feels like this came in just to bring in `RedisConfig.groovy` unchanged.

I don't think there is actually that much in there once the unneeded bits get cut out that wouldn't be too hard to port over to java (aside from the one `//thx groovy` where it's reading a private field)